tests/integration/cqlengine/test_timestamp.py

```diff
@@ -47,7 +47,7 @@ def test_batch_is_included(self):
             with BatchQuery(timestamp=timedelta(seconds=30)) as b:
                 TestTimestampModel.batch(b).create(count=1)
 
-        "USING TIMESTAMP".should.be.within(m.call_args[0][0].query_string)
```

Comment:
I would recommend switching to pytest regular asserts, and not using a 3rd party for those.

Reply:
Let's have an another PR on that - https://github.com/scylladb/python-driver/pull/592

Reply:
removed from the PR.